### PR TITLE
changed colors, fonts, in the news page

### PR DESCRIPTION
**Pull request by karindalziel**

### CODE REVIEW

File: app/assets/stylesheets/bootstrap-variables.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $bootstrap-sass-asset-helper: false !default;
 //
-// Variables
+// Variables
```
Comment (karindalziel) [inline app/assets/stylesheets/bootstrap-variables.scss:3]: Karin remove this

File: app/assets/stylesheets/bootstrap-variables.scss [hunk 2]
```diff
@@ -3,30 +3,27 @@ $bootstrap-sass-asset-helper: false !default;
 // Variables
 // --------------------------------------------------
```
Comment (karindalziel) [inline app/assets/stylesheets/bootstrap-variables.scss:5]: I moved these to the _fonts.html.erb template

File: app/assets/stylesheets/bootstrap-variables.scss [hunk 3]
```diff
@@ -3,30 +3,27 @@ $bootstrap-sass-asset-helper: false !default;
...
 
-// @import url('https://fonts.googleapis.com/css?family=Catamaran:300,900|Hepta+Slab:300,900|Josefin+Sans:300,700|Noto+Serif+JP:200,900|Podkova:400,800|Rokkitt:300,900|Zilla+Slab:300,700&display=swap');
- @import url('https://fonts.googleapis.com/css2?family=Crimson+Text:ital@0;1&family=Nunito+Sans:wght@400&display=swap');
-
 //== Colors
```
Comment (karindalziel) [inline app/assets/stylesheets/bootstrap-variables.scss:6]: I move the font call to the _fonts.html.erb template

File: app/views/layouts/body/_title.html.erb [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<div class="container site-title-container">
+  <span class="site-title-container01"></span>
```
Comment (karindalziel) [inline app/views/layouts/body/_title.html.erb:2]: extra spans are there in order to make background colors (different on each side) stretch across all the way

File: app/views/inthenews/home.html.erb [hunk 5]
```diff
@@ -1,15 +1,31 @@
...
-<div class="container-fluid home_container">
+</div>
+<div class="inthenews_background">
+  <div class="container lead">
+    <p>African Poetry and Poetics in the News  explores how depictions of African poetry in the news media have influenced and shaped perceptions of what African poetry is in the 21st century. It facilitates and supports current research relating to Poetry and Revolution, Race and Poetry, and Poetry and Migration as it relates to the African continent and cultures.</p>
+  </div>
+</div>
+<div class="container">
+
+<div class="container-fluid home_container inthenews_container">
```
Comment (techgique) [inline app/views/inthenews/home.html.erb:11]: Can we look at indenting the content inside of these divs to keep the alignment below consistent?

File: app/views/inthenews/home.html.erb [hunk 6]
```diff
@@ -33,10 +49,6 @@
         </div>
       <% end %>
     </div>
-
-  </div>
-
-  <div class="col-md-6">
-    <%= render "partials/map_africa" %>
+    <%# render "partials/map_africa" %>
```
Comment (techgique) [inline app/views/inthenews/home.html.erb:52]: Can we remove this since it should be trivial to add back by copying from the other page template?

**Review by techgique — CHANGES_REQUESTED** [review]
Just a couple questions about some changes

**Pull request merged**